lib/engine/game/g_1846.rb

```diff
@@ -134,10 +134,13 @@ def setup
       end
 
       def remove_from_group!(group, entities)
-        remove = group.sort_by { rand }.take([5 - @players.size, 2].min)
-        @log << "Removing #{remove.join(', ')}"
+        removals = group.sort_by { rand }.take([5 - @players.size, 2].min)
```

Comment:
this doesn't do what you think it does, sort_by.take returns an array

an empty array is truthy in ruby unlike in python

ruby only false and nil are falsey

you need to do return if removals.empty?

Reply:
Ah, thanks !